data/korath/korath variants.txt

```diff
@@ -133,13 +133,17 @@ ship "Rano'erek" "Rano'erek (Shunt)"
 
 ship "Rano'erek" "Rano'erek (Ground Assault)"
 	"crew" 449
+	add attributes
+		"bunks" 92
+		"mass" 115
+		"cargo space" -120
+		"outfit space" -370
+		"heat dissipation" -0.04
```

Comment:
This seems like a pretty inefficient refit to me.  If this variant was purpose-built for carrying troops, rather than simply being refitted, I'd expect it'd do away with the inefficiencies of modular outfit expansions and bunk rooms.
```suggestion
		"outfit space" -248
```